site/src/router.tsx

```diff
@@ -427,22 +432,32 @@ export const router = createBrowserRouter(
 					</Route>
 
 					<Route path="/deployment" element={<ManagementSettingsLayout />}>
-						<Route path="general" element={<GeneralSettingsPage />} />
-						<Route path="licenses" element={<LicensesSettingsPage />} />
-						<Route path="licenses/add" element={<AddNewLicensePage />} />
-						<Route path="security" element={<SecuritySettingsPage />} />
-						<Route
-							path="observability"
-							element={<ObservabilitySettingsPage />}
-						/>
-						<Route path="appearance" element={<AppearanceSettingsPage />} />
-						<Route path="network" element={<NetworkSettingsPage />} />
-						<Route path="userauth" element={<UserAuthSettingsPage />} />
-						<Route
-							path="external-auth"
-							element={<ExternalAuthSettingsPage />}
-						/>
+						<Route element={<DeploymentSettingsProvider />}>
```

Comment:
I don't have strong opinions either way, but I felt it could help to move any components that don't rely on `DeploymentSettingsProvider` outside of it